bindings/python/native/iota_client/_node_indexr_api.py

```diff
@@ -0,0 +1,60 @@
+from iota_client.common import send_message_routine, call_client_method
+
+
+class NodeIndexerAPI():
+
+    @send_message_routine
+    def output_ids(self, query_parameters):
+        """Fetch output IDs.
+        """
+        return call_client_method('OutputIds', {
+            'query_parameters': query_parameters
+        })
+
+    @send_message_routine
+    def aliases_out_ids(self, query_parameters):
+        """Fetch aliases output IDs.
+        """
+        return call_client_method('AliasesOutputIds', {
+            'query_parameters': query_parameters
+        })
+
+    @send_message_routine
+    def alias_out_ids(self, alias_id):
+        """Fetch alias output IDs.
+        """
+        return call_client_method('AliasOutputIds', {
+            'alias_id': alias_id
+        })
+
+    @send_message_routine
+    def nfts_output_ids(self, query_parameters):
+        """Fetch NFTs output IDs.
+        """
+        return call_client_method('NftsOutputIds', {
+            'query_parameters': query_parameters
+        })
+
+    @send_message_routine
+    def nfts_output_id(self, nft_id):
+        """Fetch NFT output IDs.
+        """
+        return call_client_method('NftOutputIds', {
+            'nft_id': nft_id
+        })
+
+    @send_message_routine
+    def foundries_output_ids(self, query_parameters):
+        """Fetch Foundries Output IDs.
+        """
+        return call_client_method('FoundriesOutputIds', {
+            'query_parameters': query_parameters
+        })
+
+    @send_message_routine
+    def foundry_output_ids(self, foundry_id):
+        """Fetch Foundry Output IDs.
```

Comment:
`foundry_output_id` and
```suggestion
        """Fetch Foundry Output ID.
```